parsing/location.ml

```diff
@@ -897,6 +895,15 @@ let alert ?(def = none) ?(use = none) ~kind loc message =
 let deprecated ?def ?use loc message =
   alert ?def ?use ~kind:"deprecated" loc message
 
+let auto_include_alert lib =
+  let message = Printf.sprintf "\
+    OCaml's lib directory layout changed in 5.0. The %s subdirectory has been \
+    automatically added to the search path, but you should add -I +%s to the \
+    command-line to silence this alert." lib lib in
```

Comment:
I suspect most project use dune these days, maybe this is slightly better advice?

```suggestion
    OCaml's lib directory layout changed in 5.0. The %s subdirectory has been \
    automatically added to the search path, but you should add -I +%s to the \
    command-line (or add %s to the list of libraries in the dune file) to silence \
    this alert." lib lib lib in
```

Reply:
Hmm - I like the idea, but I'm loathe to make it completely Dune-specific. How about

```suggestion
    OCaml's lib directory layout changed in 5.0. The %s subdirectory has been \
    automatically added to the search path, but you should add -I +%s to the \
    command-line to silence this alert (e.g. by adding %s to the list of \
    libraries in your dune file, or adding use_%s to your _tags file for \
    ocamlbuild, or using -package %s for ocamlfind)." lib lib lib lib lib in
```

Reply:
In term of formatting, since the alert reporting is using the message string directly, the line breaks need be added directly to the message. One option would beto add  `\n` line break directly, the other would be to switch to `Format.pp_print_text` with 

```ocaml
Format.asprintf "%a" Format.pp_print_text  
  "OCaml's lib directory layout changed in 5.0...."
```

Reply:
In term of formatting, since the alert reporting is using the message string directly, the line breaks need be added directly to the message. One option would beto add  `\n` line break directly, the other would be to switch to `Format.pp_print_text` with 

```ocaml
Format.asprintf "%a" Format.pp_print_text  
  "OCaml's lib directory layout changed in 5.0...."
```